### ISSUE TITLE
UndefVarError: chm_scale! not defined

### ISSUE THREAD

**Issue by nilshg** [dmbates/MixedModels.jl#32]
I'm getting this error on Julia 0.4.0-rc1:

````
julia> using MixedModels
ERROR: LoadError: UndefVarError: chm_scale! not defined
  in include at boot.jl:260
  in include_from_node1 at loading.jl:271
  in include at boot.jl:260
  in include_from_node1 at loading.jl:271
  in require at loading.jl:210
while loading [...]\MixedModels\src\utils.jl, in expression starting on line 39
while loading [...]\MixedModels\src\MixedModels.jl, in expression starting on line 44
````

I didn't do anything but install a fresh version of the release candidate and call `Pkg.add("MixedModels")`.

**Comment by protogeezer** [comment]
Yes, I'm seeing the same thing... with Julia 0.4.0-rc1+7 and MixedModels 0.3.22

**Comment by dmbates** [comment]
I'll make a new release tomorrow (Sept. 16) to fix this.

**Comment by dmbates** [comment]
The new release is out.  @nilshg @protogeezer Have you successfully updated?

**Comment by nilshg** [comment]
Yes, problem solved. Thanks!

**Comment by protogeezer** [comment]
Greetings.

I was able to update (on OS X 10.10.5) after hitting a pothole: Xcode 7 only includes the 10.11 SDK. It does not include the 10.9 nor 10.10 SDKs that were included with Xcode 6.4. 

After manually copying the 10.9 SDK from a machine that doesn’t have Xcode 7, I was able to update and issue a using command successfully.

Thanks for the quick turnaround. 

Stephen
